frontend/src/app/applications/permit-application-view/temp-outfitter-details.component.html

```diff
@@ -48,7 +48,7 @@ <h2 class="app-item-label padding-right-1">Applicant information</h2>
         <span *ngIf="application.applicantInfo.fax.extension"> ext. {{ application.applicantInfo.fax.extension }}</span>
       </span>
       <span class="display: block" *ngIf="application.applicantInfo.website">
-        Website: <a href="{{application.applicantInfo.website | UrlPipe }}" target="_blank">{{application.applicantInfo.website}}</a>
+        Website: <a title="WARNING: URL provided by applicant will open in new window." href="{{application.applicantInfo.website | UrlPipe }}" target="_blank">{{application.applicantInfo.website}}</a>
```

Comment:
```suggestion
        Website: <a href="{{application.applicantInfo.website | UrlPipe }}" target="_blank">{{application.applicantInfo.website}}</a> (WARNING: URL provided by applicant will open in new window.)
```

From Julie's suggestion in the previous PR.

Reply:
Don't forget this one. 😄

Reply:
@mgwalker - Hey Greg, how does it look? This should be what you would see on the applications details page now.

![image](https://user-images.githubusercontent.com/23467847/96404897-a9234580-11a1-11eb-9407-3e562184ed3e.png)

And here's the code:

![image](https://user-images.githubusercontent.com/23467847/96404940-c35d2380-11a1-11eb-87db-fb75576c2384.png)

Reply:
That looks great! There's also one for just "Website" (starting at line 51 in the HTML).